app/src/main/java/ch/blinkenlights/android/vanilla/SlidingPlaybackActivity.java

```diff
@@ -85,6 +85,7 @@ protected void bindControlButtons() {
 		mSeekBar = (SeekBar)findViewById(R.id.seek_bar);
 		mSeekBar.setMax(1000);
 		mSeekBar.setOnSeekBarChangeListener(this);
+		ThemeHelper.setAccentColor(this);
```

Comment:
Why would we do this in bindControlButtons?

Reply:
The bindControlButtons is the first method that finds the seekbar. Normally i would do it in the onCreate method, but there was none. I could create it and move it there, though.